{45293: 0}
{45293: 0.007864763960242271}
{45293: 0.015263373963534832}
{45293: 0.02593219093978405}
{45293: 0.036853302735835314}
{45293: 0.060937832575291395}
{45293: 0.07222928758710623}
{45293: 0.11514040641486645}
{45293: 0.12379572121426463}
{45293: 0.1609346093609929}
{45293: 0.1722378977574408}
{45293: 0.21003393083810806}
{45293: 0.22588907601311803}
{45293: 0.27296207193285227}
{45293: 0.28939873399212956}
{45293: 0.7940424508415163}
{45293: 0.9255954260006547}
{45293: 1.2998092551715672}
{45293: 1.3073354652151465}
{45293: 9408.25917890016}
{45293: 21269.002383596264}
{45293: 21269.217833682895}
{45293: 21269.29084449075}
{45293: 21269.51676863432}
{45293: 21269.73491955409}
{45293: 21269.988191301003}
{45293: 21270.276288774796}
{45293: 21270.501956115477}
{45293: 21270.58560368372}
{45293: 21270.820950879715}
{45293: 21270.82470866898}
{45293: 21271.889611436985}
{45293: 21272.176379465032}
{45293: 21273.254759795964}
Traceback (most recent call last):
  File "ra.py", line 41, in <module>
    vm.verify_full_state()
  File "/ssd//tools/git/RA/vm.py", line 285, in verify_full_state
    self.run()
  File "/ssd//tools/git/RA/vm.py", line 845, in run
    ret = func(s, *funcarg)
  File "/ssd//tools/git/RA/vm.py", line 1438, in op_calldataload
    raise DevelopmentErorr()
exceptions.DevelopmentErorr: Implementation or design error. 
